### ISSUE TITLE
Articles list is blank after opening image link in browser

### ISSUE THREAD

**Issue by crankygeek** [Ranchero-Software/NetNewsWire#3641]
NNW 6.1 (6106) on iPadOS 15.6

When "Open Links in NetNewsWire" is disabled, opening images in a feed that are also links can result in an empty articles list after returning to NNW from the browser and closing the full screen image. See attached video.

I found this with Gus' feed: http://shapeof.com/rss.xml. I’m subscribed via Feedbin, but I don’t believe that has any bearing on this issue.

This seems to be triggered by a combination of the images being links, rather than just simply embedded in the post, and the full screen image view.

Steps to reproduce

1. In NNW settings, turn off “Open Links in NetNewsWire”
2. Subscribe to `http://shapeof.com/rss.xml`
3. View the post “A Geeky Filter for Acorn Arriving in 7.3"
4. Scroll down to find one of the images in the post and tap on it to open it
5. Safari should launch and display the image
6. Return to NNW and observe the image is displayed in full screen
7. Tap X to dismiss the image
8. Observe the Articles list is blank

https://user-images.githubusercontent.com/1638278/182484461-6cc9aaaa-a538-4130-96d5-6f28d2f306a3.mp4

**Comment by vincode-io** [comment]
I think we should probably disable image zooming on any image that is contained in a link.

**Comment by crankygeek** [comment]
This may be overkill, but would you want to show some kind of indicator on images in links to show that opening it will cause a different behavior?

Part of the problem with something like that is that it would only need to be visible if open in NNW is disabled.

**Comment by vincode-io** [comment]
I don't think we need an indicator. I also don't think this is specific to "Open in NNW". I'm not trying to hide the bug. I just think the current behavior is actually wrong. If the author determines that an image is a link, we shouldn't be zooming it anyway. Maybe I'm wrong, but I think image links are different than images designed to explain or visually show a topic.

**Comment by crankygeek** [comment]
I understand what you mean. I guess I was just thinking that since there's no visual indication within the display of a post, if a user is used to images getting zoomed, but then all of a sudden some images are opening in a browser view/external browser, it could get confusing.

Honestly, it's it's kind of an edge case. It's probably best to make the change as you suggest and wait to see if there's any feedback about it.

**Comment by vincode-io** [comment]
Agreed. We can always add a little link overlay in the corner of the picture later if it turns into a thing.
